src/containers/Login.vue

```diff
@@ -379,7 +336,7 @@ input {
 }
 .email-button {
   border: 0;
-  background-color: $emGreen;
+  background-color: #539d9b;
```

Comment:
This hex color seems to only be used on this login button right now, which makes it seem like it might be the wrong color on Figma... @einc is this the color we want the landing button to be?